.dockerignore

```diff
@@ -0,0 +1,5 @@
+.insightface
```

Comment:
Thanks but please revert this...

Reply:
I think you misunderstand what `.dockerignore` does. It has to be there to prevent Docker from uploading the directory `.insightface` (and other directories) to the Docker daemon when building the image.

All this file does is it prevents an unnecessary upload of about 1GB of data that is not required for building the Docker image.

Here is the documentation for `.dockerignore`:
https://docs.docker.com/engine/reference/builder/#dockerignore-file